### PR TITLE
Add GW REST APIs section

### PR DESCRIPTION
**Pull request by taban03**
Signed-off-by: at670475 <andrea.tabone@broadcom.com>

### Your checklist for this pull request
🚨Please review the [guidelines for contributing](../docs/contribute/contributing.md) to this repository.

- [ ] If the changes in this PR is part of the next future release, make this pull request against the **docs-staging** branch which will be published at the next release boundary. If the changes in this PR are part of the current release, use the default base branch, **master**. For more information about branches, see https://github.com/zowe/docs-site/tree/master#understanding-the-doc-branches. 
      
- [ ] If this PR relates to GitHub issues in `docs-site` or other repositories, please list in Description, prefixed with **close**, **fix** or **resolve** keywords.

### Description (including links to related git issues)
Please describe your pull request.

:heart:Thank you!

### CODE REVIEW

File: docs/user-guide/api-mediation/api-mediation-jwt-token-refresh.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+- Cookie named `apimlAuthenticationToken`.
+- Bearer authentication
+
+**Notes:**
+
+- The endpoint is disabled by default. For more information, see [Enable JWT token endpoint](api-gateway-configuration.md#enable-jwt-token-refresh-endpoint).
+- The endpoint is protected by a client certificate.
+
+  For more information, see the OpenAPI documentation of the API Mediation Layer in the API Catalog.
```
Comment (janan07) [inline docs/user-guide/api-mediation/api-mediation-jwt-token-refresh.md:15]: Is there a specific link?

Comment (taban03) [reply docs/user-guide/api-mediation/api-mediation-jwt-token-refresh.md:15]: No, the OpenAPI documentation is present in the catalog, not in zowe doc

File: docs/user-guide/api-mediation/api-mediation-update-password.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+* [Change password with SAF provider](#change-password-with-saf-provider)
+* [Change password with z/OSMF provider](#change-password-with-zosmf-provider)
+
+
+## Change password with SAF provider
+
+Update the user password using the SAF Authentication provider. To use this functionality, add the parameter `newPassword` on the login endpoint `/gateway/api/v1/auth/login`. The Gateway service returns a valid JWT with the response code `204` as a result of successful password change. The user is then authenticated and can consume APIs through the Gateway. If it is not possible to change the password for any reason, the response code is `401`.
+
+This feature is also available in the API Catalog.
```
[the remark this answers is not in the document]
Comment (taban03) [reply docs/user-guide/api-mediation/api-mediation-update-password.md:13]: yes this is changing the pw using saf provider

**Review by janan07 — APPROVED** [review]
Reviewed and edited by tech info

File: docs/user-guide/api-mediation/api-mediation-jwt-token-refresh.md (hunk 1 above)
Comment (balhar-jakub) [reply docs/user-guide/api-mediation/api-mediation-jwt-token-refresh.md:15]: @taban03 The documentation is involved in the Zowe doc. https://docs.zowe.org/stable/appendix/zowe-api-reference

**Pull request merged**